spec/support/pages/bank_entries_page.rb

```diff
@@ -1,31 +1,26 @@
 class BankEntriesPage < BasePage
   class AccountEntrySection < SitePrism::Section
-    element :amount,        ".table__cell--amount"
-    element :amount_field,  ".table__cell--amount input"
-    element :account,       ".table__cell--account"
-    element :account_field, ".table__cell--account select"
+    element :amount,        '.table__cell--amount'
+    element :amount_field,  '.table__cell--amount input'
+    element :account,       '.table__cell--account'
+    element :account_field, '.table__cell--account select'
   end
 
   class BankEntrySection < SitePrism::Section
-    sections :account_entries, AccountEntrySection, ".table__row[ng-repeat='accountEntry in entry.accountEntries']"
-    element :close_button,  "button", text: 'close'
-    element :save_button,   "button", text: 'save'
-    element :cancel_button, "button", text: 'cancel'
-    element :saving_message, "span",  text: 'saving...'
+    sections :account_entries, AccountEntrySection,
+      ".table__row[ng-repeat='accountEntry in entry.accountEntries']"
+    element :close_button,  'button', text: 'close'
+    element :save_button,   'button', text: 'save'
+    element :cancel_button, 'button', text: 'cancel'
+    element :saving_message, 'span',  text: 'saving...'
 
-    def click
-      root_element.click
-    end
-
-    def text
-      root_element.text
-    end
+    delegate :click, :text, to: :root_element
   end
 
   set_url '/v3#/entries'
-  set_url_matcher /\/v3#\/entries$/
+  set_url_matcher %r{/v3#/entries$}
 
-  sections :bank_entries, BankEntrySection, ".table__rows"
+  sections :bank_entries, BankEntrySection, '.table__rows'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.